tests/unit/channel_test.py

```diff
@@ -237,7 +237,7 @@ def test_buffer_frame(self):
     def test_process_frames_when_no_frames(self):
         # Not that this should ever happen, but to be sure
         c = Channel(None, None, {})
-        stub(c.dispatch)
+        expect(c.dispatch).times(0)
```

Comment:
I wasn't aware of that initially, so I thought that it wasn't testing anything; sort of like patching something in the mock library, but then not checking call count.

However, once I learned the semantics of stub better, I realized that I still liked using `times(0)` in such cases, because it's explicit vs. implicit. I'd be happy to change that back. Please let me know.